packages/components/src/hooks/useDgsData/useDgsData.ts

```diff
@@ -17,11 +17,10 @@ export const useDgsData = ({
   resourceId,
   limit,
   offset,
-  fields,
   filters,
   sort,
 }: UseDgsData) => {
-  const [isLoading, setIsLoading] = useState(true)
+  const [isLoading, setIsLoading] = useState(false)
```

Comment:
question(non-blocking): hmm for my understanding, how come we need to set this to `false` as an initial value ah, I thought starting with `true` makes sense to always show the loading state first

Reply:
true, makes sense. updated https://github.com/opengovsg/isomer/pull/1579/commits/a660d13b5d67a143cb1fb0508dec52b1e5610806